src/main/java/com/cgi/ipsen5/Repository/LocationRepository.java

```diff
@@ -11,6 +11,7 @@
 public interface LocationRepository extends JpaRepository<Location, UUID> {
     List<Location> findAllByWingId(UUID wingId);
     List<Floor> findAllByWing_FloorId(UUID id);
+    List<Location> findAllByWing_Floor_BuildingId(UUID buildingId);
```

Comment:
hoezo gebruik je hier snake case?

Reply:
Dat is hoe via spring boot jpa de data wordt opgehaald. Geen idee of het ook anders opgeschreven kan worden eigenlijk

Reply:
Als het goed is kun je de streepjes negeren en gewoon doorgaan zonder streepjes